static/js/containers/DashboardPage.js

```diff
@@ -550,6 +550,10 @@ class DashboardPage extends React.Component {
     );
   }
 
+  openLinkInNewTab = (url: ?string): void => {
+    window.open(url, '_blank');
+  }
```

Comment:
I just tried in the node repl and I was able to do it, so there shouldn't be any problem doing it in our tests. You just do something like this in the `beforeEach` method for the relevant tests:

```js
let windowOpenStub = sandbox.stub(window, 'open');
```

then you can do the test and check that `windowOpenStub` is called with the arguments you expect.